src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -538,6 +538,14 @@ xrtSyncBOAIE(xrtGraphHandle ghdl, unsigned int bo, const char *dmaID, enum xclBO
   graph->sync_bo(bo, dmaID, dir, size, offset);
 }
 
+void
+xrtResetAieArray(xclDeviceHandle handle)
+{
+  auto device = xrt_core::get_userpf_device(handle);
+  XAieLib_NpiAieArrayReset(XAIE_RESETENABLE);
+  XAieLib_NpiAieArrayReset(XAIE_RESETDISABLE);
```

Comment:
Doesn't look the device is used here?  Can you remove line 544?

Reply:
My intention here is to do a handle check. The caller should open the device before operate on it. Do you think it is necessary? Or do we have a better way to do it?

Reply:
Ok. intention is fine, makes sense.  Maybe comment to that effect and maybe don't save the return value.

Reply:
Done. Thanks!